lib/node_modules/@stdlib/math/base/complex/divide/benchmark/cdiv_smith.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+
+// MAIN //
+
+/**
+* Divides two complex numbers.
```

Comment:
I've resolved this by adding a benchmark of Smith's method. (It seems about 25% faster.)